mappings/net/minecraft/client/gui/hud/chat/MessageStorage.mapping

```diff
@@ -0,0 +1,73 @@
+CLASS net/minecraft/unmapped/C_scihbqqa net/minecraft/client/gui/hud/chat/MessageStorage
+	FIELD f_eyexyhru length I
+	FIELD f_krzhrpzx offset I
+	FIELD f_kwqbgwyl DEFAULT_SIZE I
+	FIELD f_wnubqcju storage [Ljava/lang/Object;
+	METHOD <init> (I)V
+		ARG 1 size
+	METHOD add add (ILjava/lang/Object;)V
+		ARG 1 index
+		ARG 2 object
+	METHOD addFirst addFirst (Ljava/lang/Object;)V
+		ARG 1 object
+	METHOD addLast addLast (Ljava/lang/Object;)V
+		ARG 1 object
```

Comment:
I feel like these `object`s could be a bit more specific

Reply:
would you prefer `element`? That's what `java.util.AbstractList` uses.

Reply:
That sounds good